airbyte-integrations/connectors/source-amazon-ads/streams/attribution_report_performance_creative.patch.json

```diff
@@ -0,0 +1,9 @@
+{
+    "required": [
+        "adGroupId"
+    ],
+    "type": ["object"],
+    "properties": {
+        
+    }
+}
```

Comment:
These files are not necessary since `airbyte-to-flow` will automatically mark the keys as required properties

Reply:
Actually, this is not only for setting the required parameter, it also needs the type: [object] only, else it was taking null as the default was set to [object, null]

Reply:
Removed the "required" attribute, "type" is needed here I have checked that.